data/chiomavera.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Nkanmuo Vera Chioma",
+  "type": "personal",
+  "bio": "**A Frontend Developer** who enjoys creating responsive and functional Web Applications that are accessible to all.",
+    "tags": [
+    "Open Source",
+    "React.js",
+    "JavaScript",
+    "Tailwind",
+    "Bootstrap",
+    "CSS",
+    "HTML"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/nkanmuo_vera" },
+    { "icon": "FaGithub", "url": "https://github.com/chiomavera" },
+    { "icon": "FaLinkedIn", "url": "https://www.instagram.com/pradumnasaraf" }
```

Comment:
```diff
+ "icon": "FaLinkedin"
```
Also, correct the URL to a valid Linkedin URL

Reply:
Thanks, @SandeepKrSuman for reviewing my PR.
I have made the requested changes